### DISCUSSION TITLE
Connection Pooling from Serverless Compute Platforms

### DISCUSSION THREAD

**Discussion by DanielSanchez97** [neondatabase/neon#1888] [Q&A] [answered]
This looks like a terrific product! I am working on a Python based Serverless framework (https://cdevframework.io), and one of the resources that we support is the Aws Aurora Serverless Postgress DB. One of the benefits of this resource is that is designed with the idea of handling many connections from a Serverless Compute Platform like Aws Lambda via SQL over HTTPS. I am just wondering if your platform has optimizations around handling a large amount of connections from a Serverless Compute platform like Aws Lambda?

**Comment by hlinnaka** [comment, marked as answer]
No, there are no optimizations for that yet. We'll need to address that in the future I'm sure.

**Reply by DanielSanchez97** [reply]
Thanks for the quick reply! If yall do get around to that feature let me know!
